front/pages/api/v1/w/[wId]/apps/[aId]/runs/index.ts

```diff
@@ -503,3 +505,25 @@ async function handler(
 }
 
 export default withLogging(handler);
+
+async function getGroupsForKey(
```

Comment:
Why don't go all the way and get an actual auth for the user instead of stitching in groups? Should be useful for future us no?

Reply:
Updated to only deal with an auth object.